src/modules/getShows.js

```diff
@@ -0,0 +1,60 @@
+const myURL = 'https://api.tvmaze.com/shows';
+
+const getShows = async () => {
+  const response = await fetch(`${myURL}`);
+  let datas = await response.json();
+  datas = datas.slice(0, 6);
```

Comment:
[OPTIONAL} Kindly remove this line, it's better to show all the elements and no limit them.

Reply:
They are too many and we will need Pagination to show all of them. Please let us display around 24 of them

Reply:
Ok, let's display only 24 elements